src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.motors.PIDSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+@RobotMapFor("Gaston")
+public class GastonMap extends RobotMap {
+    private final DigitalInput outsideLimit = new DigitalInput(4);
+    private final DigitalInput insideLimit = new DigitalInput(5);
+
+    // initializes relative encoder and pid controller, we don't need the encoder rn
+
+    private PIDSparkMax deploymentMotor = new PIDSparkMax(2, MotorType.kBrushless);
+
+    // private RelativeEncoder deploymentEncoder =
+    // deploymentMotor.getEncoder().getRaw();
+    private final SparkMaxPIDController deploymentPidController = deploymentMotor.getPidController();
+
+    public IntakeMap getIntakeMap() {
+        // PID coefficients
+
+        double P = 5e-5;
+        double I = 1e-6;
```

Comment:
These can be 0 for now, unless you've calculated these based on anything specific.